src/App.js

```diff
@@ -1,25 +1,62 @@
-import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
+import contacts from "./contacts.json";
 
 function App() {
+
+  const [contactsArr, setContact] = useState(contacts.slice(0, 5));
+  const addRandomContact = () => {
+    const randomContact = contacts[Math.floor(Math.random() * contacts.length)];
+    setContact((prevContacts) => {
+      const newContacts = [...prevContacts, randomContact]
+      return newContacts
+    })
+  }
```

Comment:
Last step for this lab would be to display list without repeated contacts :) 

Clue: you might want to create new array that removes contacts that already being displayed, and get random contact from that new list

Reply:
Right !!! I completely forgot about this :/